### PR TITLE
Travis and Coveralls YAMLs, add .gitignore. Resolves #2

### LINKED ISSUE #2: Set up Travis and Coveralls for cromwell-backend

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

**Comment by mcovarr** [comment]
Reviewers: @francares and @scottfrazer

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea
+target
```
Comment (francares) [inline .gitignore:2]: You may want to add:
*.*~
*.sc

**Comment by francares** [comment]
Looks OK to me.

File: .gitignore (hunk 1 above)
Comment (mcovarr) [inline .gitignore:2]: What are *.sc files?

Comment (francares) [inline .gitignore:2]: They are Scala worksheets... We make use a lot of this feature.

Comment (mcovarr) [inline .gitignore:2]: Ok, added.

Comment (francares) [inline .gitignore:2]: I just realized I didn't escape asterisks in my first comment. Please take a look to it.

Comment (mcovarr) [inline .gitignore:2]: Ha, makes more sense now.  :smile:

**Comment by scottfrazer** [comment]
:+1:

**Pull request merged**